### PR TITLE
Added explanation of surface tag list assets

### PR DESCRIPTION
**Pull request by cgalvan**
## Change summary

Added explanation of `Surface Tag List Assets` and how to create new custom surface tags. Also updated all component pages that referenced surface tags to point to this page.

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

Signed-off-by: Chris Galvan <chgalvan@amazon.com>

### CODE REVIEW

File: content/docs/user-guide/gems/reference/environment/surface-data.md [hunk 1]
```diff
@@ -6,4 +6,19 @@ weight: 400
 toc: true
 ---
 
-The Surface Data Gem enables surfaces such as terrain or meshes to emit signals or tags, that communicate its surface type. These signals and tags have various uses, such as creating inclusion and exclusion areas for vegetation growth.
+The Surface Data Gem enables surfaces such as terrain or meshes to emit signals or tags, that communicate its surface type. These signals and tags have various uses, such as creating inclusion and exclusion areas for vegetation growth or terrain.
```
Comment (micronAMZN) [inline content/docs/user-guide/gems/reference/environment/surface-data.md:9]: minor suggestion to add additional information.

```suggestion
The Surface Data Gem enables surfaces and meshes to emit signals or tags that communicate various attributes. These signals and tags have various uses such as creating inclusion and exclusion areas, aligning and positioning objects such as vegetation, and applying physics materials.
```

File: content/docs/user-guide/gems/reference/environment/surface-data.md [hunk 2]
```diff
@@ -6,4 +6,19 @@ weight: 400
...
-The Surface Data Gem enables surfaces such as terrain or meshes to emit signals or tags, that communicate its surface type. These signals and tags have various uses, such as creating inclusion and exclusion areas for vegetation growth.
+The Surface Data Gem enables surfaces such as terrain or meshes to emit signals or tags, that communicate its surface type. These signals and tags have various uses, such as creating inclusion and exclusion areas for vegetation growth or terrain.
+
+## Adding Surface Tag Names
+
+New **Surface Tag** names can be added by creating a new **Surface Tag Name List** asset. This can be done with the following steps:
+
+1. Open the **Asset Editor** from the **Tools** menu.
+1. Select **File** -> **New** -> **Surface Tag Name List**.
+1. Click the **+** button to the right of the **Surface Tag Name List** row, which will add custom **Surface Tag** name elements.
```
Comment (micronAMZN) [inline content/docs/user-guide/gems/reference/environment/surface-data.md:17]: Adding an inline icon

```suggestion
1. Click the {{< icon "add.svg" >}} **Add** button to the right of the **Surface Tag Name List** row, which adds a custom **Surface Tag** name element.

File: content/docs/user-guide/gems/reference/environment/surface-data.md [hunk 3]
```diff
@@ -6,4 +6,19 @@ weight: 400
...
+1. Fill in the **Surface Tag** name elements with tag names you would like to use.
+
+![Adding surface tag names.](/images/user-guide/gems/reference/surface-data/surface-data-custom-tag.png)
+
+Once you save your **Surface Tag Name List** asset, the tag(s) you added will now be available in any **Surface Tag** data field.
```
Comment (micronAMZN) [inline content/docs/user-guide/gems/reference/environment/surface-data.md:22]: Suggestion for style

```suggestion
Once you save the **Surface Tag Name List** asset, the tag(s) you added are available in any **Surface Tag** data field.
```

**Review by micronAMZN — APPROVED** [review]
Three minor suggestions to improve the original text in the surface data topic.

**Review by chanmosq — APPROVED** [review]

File: content/docs/user-guide/gems/reference/environment/surface-data.md (hunk 1 above)
Comment (cgalvan) [reply content/docs/user-guide/gems/reference/environment/surface-data.md:9]: Sounds good!

File: content/docs/user-guide/gems/reference/environment/surface-data.md (hunk 3 above)
Comment (cgalvan) [reply content/docs/user-guide/gems/reference/environment/surface-data.md:22]: Done!

File: content/docs/user-guide/gems/reference/environment/surface-data.md (hunk 2 above)
Comment (cgalvan) [reply content/docs/user-guide/gems/reference/environment/surface-data.md:17]: Nice!

**Pull request merged**